.eslintrc.json

```diff
@@ -11,9 +11,10 @@
         "asyncArrow": "always",
         "named": "never"
     }],
-    "jsx-quotes": ["error", "prefer-double"]
+    "jsx-quotes": ["error", "prefer-double"],
+    "react/jsx-fragments": "off"
```

Comment:
@lotrien I turned off only one default rule added after eslint upgrade. feel free to disable/enable other rules